README.md

```diff
@@ -1,4 +1,9 @@
-Travis-CI: [![Build Status](https://travis-ci.org/ipa320/cob_android.svg?branch=indigo_dev)](https://travis-ci.org/ipa320/cob_android)
-
 cob_android
 ===========
+
+Travis-CI: [![Build Status](https://travis-ci.org/ipa320/cob_android.svg?branch=indigo_dev)](https://travis-ci.org/ipa320/cob_android)
+
+ROS-Distro-Support:
+ - ROS Indigo Igloo (branch: `indigo_dev`)
```

Comment:
@ipa-bnm @ipa-mdl @ipa-fmw
How do you like this? https://github.com/ipa-fxm/cob_android/blob/indigo_kinetic_travis/README.md